src/core/3p-frame-messaging.js

```diff
@@ -142,16 +142,16 @@ export function isAmpMessage(message) {
 }
```

Comment:
Nit: Since we're here, we can reimplement `isAmpMessage`:

```js
export function isAmpMessage(message) {
  return (
    typeof message == 'string' &&
    message.startsWith(AMP_MESSAGE_PREFIX) &&
    message.indexOf('{') != -1
  );
}
```

Reply:
Done